tools/README.md

```diff
@@ -53,17 +53,17 @@ Developer tools which you might find useful:
   can conveniently run diffs on them when working on code-generation.
   (See also [generated_dirs.txt](generated_dirs.txt) which
   specifies the list of directories with generated files.)
-* [mypy_wrapper.py](mypy_wrapper.py) - Run `mypy` on a single file using the
+* [linter/mypy_wrapper.py](linter/mypy_wrapper.py) - Run `mypy` on a single file using the
   appropriate subset of our `mypy*.ini` configs.
-* [run_shellcheck.sh](run_shellcheck.sh) - Find `*.sh` files (recursively) in
+* [linter/run_shellcheck.sh](linter/run_shellcheck.sh) - Find `*.sh` files (recursively) in
   the directories specified as arguments, and run [ShellCheck][] on all of them.
-* [test_history.py](test_history.py) - Query S3 to display history of a single
+* [stats/test_history.py](stats/test_history.py) - Query S3 to display history of a single
   test across multiple jobs over time.
-* [trailing_newlines.py](trailing_newlines.py) - Take names of UTF-8 files from
+* [linter/trailing_newlines.py](linter/trailing_newlines.py) - Take names of UTF-8 files from
   stdin, print names of nonempty files whose contents don't end in exactly one
   trailing newline, exit with status 1 if no output printed or 0 if some
   filenames were printed.
-* [translate_annotations.py](translate_annotations.py) - Read [Flake8][] or
+* [linter/translate_annotations.py](linter/translate_annotations.py) - Read [Flake8][] or
```

Comment:
for consistency, the things that are now under `linter/` or `stats/` should be in sub-bullets (similar to `shared/`, `fast_nvcc/`, and `amd_build/`)